.travis.yml

```diff
@@ -1,15 +1,37 @@
 os: linux
 dist: xenial
 language: node_js
-node_js:
-  - 10
-  - 12
 
 jobs:
   include:
-    - name: "Lint code and commit message format"
-      node_js: "10"
-      env: VALIDATE_COMMIT_MSG=true LINT=true
+    - node_js: "10"
+      script:
+        - npm run init
```

Comment:
I guess these are now running on the same filesystem?  Do we need to init/build on node N-2 if we just finished node N?

Reply:
No, these are run on different filesystems completely independently, so we need to install dependencies and run `init` on both of them.